torch/_dynamo/variables/user_defined.py

```diff
@@ -673,6 +673,12 @@ def call_function(
                 return variables.TorchCtxManagerClassVariable(
                     obj.__class__
                 ).call_function(tx, [var], kwargs)
+
+            func_src = AttrSource(self.source, "__func__") if self.source else None
```

Comment:
I believe source is required for VariableBuilder to work.  We should call `unimplemented()` if `source=None`